packages/blade/src/components/ListView/ListViewFilters.web.tsx

```diff
@@ -88,6 +88,7 @@ const ListViewFilters = ({
   const searchNameValue = searchName || searchId;
   const isMobile = useIsMobile();
   const { theme } = useTheme();
+  const showSearchInput = onSearchChange || onSearchClear || searchValuePlaceholder || searchName;
```

Comment:
Oh I missed this earlier but why do we need `onSearchClear` prop? wouldn't onSearchChange with empty value be considered as search clear?

Reply:
yes that can be considered as well. will remove this .

Reply:
yes we can use that well . will remove onSearchClear.

Reply:
onSearchClear does not get called when the value is cleared.
You can check this example: https://stackblitz.com/edit/akftnedq?file=App.tsx
We cannot remove onSearchClear.